template/.eslintrc

```diff
@@ -3,7 +3,7 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:jest/recommended",
-    "plugin:prettier/recommended"
```

Comment:
Out of curiosity, why use an `.eslintrc` file in the template, as opposed to putting it in the `package.json` like the sdk?

Reply:
I got feedback that others prefer the individual config files vs the `package.json` approach. Someday in the future I think we are going to shift the sdk to look a little more like the template projects.